i18n/en.json

```diff
@@ -3070,6 +3086,10 @@
     "id": "api.user.update_active.permissions.app_error",
     "translation": "You do not have the appropriate permissions"
   },
+  {
+    "id": "api.user.update_active.not_enable.app_error",
+    "translation": "You cannot deactivate yourself because this feature is not enable by the administrator. Please contact the Administrator."
```

Comment:
`You cannot deactivate yourself because this feature is not enabled. Please contact your System Administrator.`

Reply:
done